templates/web/base/waste/garden/subscribe_details_bin_desc.html

```diff
@@ -0,0 +1,38 @@
+<div class="waste-service-hint-banner">
+  <div class="waste-service-hint-banner__image">
+    [% IF cobrand.moniker == 'brent' %]
+      <img src="/i/waste-containers/bin-green.png" alt="">
+    [% ELSIF cobrand.moniker == 'bromley' %]
+      <img src="/i/waste-containers/bin-black-brown-lid-recycling.png" alt="">
+    [% ELSIF cobrand.moniker == 'kingston' %]
+      <img src="/i/waste-containers/bin-grey-green-lid-recycling.png" alt="">
+    [% ELSIF cobrand.moniker == 'merton' %]
+      [% image = PROCESS "waste/garden/wheelie.svg" %]
+      <span style="--primary-color: #8B5E3D;">[% image.replace('<svg', '<svg style="height:auto;max-width:100%;"') | safe %]</span>
+    [% ELSIF cobrand.moniker == 'sutton' %]
+      <img src="/i/waste-containers/bin-green-brown-lid.png" alt="">
+    [% ELSE %]
```

Comment:
Should there be some kind of fallback here, perhaps even an HTML comment explaining that there's no banner image?

Reply:
I wanted it work okay by default without showing something 'wrong', though I guess it could show any bin image and then it'd hopefully be spotted as part of testing, same as the "???" in the fallback text part, perhaps should just show a black bin as I doubt garden bins are ever that.